src/Form/index.tsx

```diff
@@ -257,21 +261,14 @@ function Form({
   // Set to trigger conditional renders on field value updates, no need to use the value itself
   const [render, setRender] = useState({ v: 1 });
 
-  const [loaders, setLoaders] = useState<Record<string, any>>({});
-  const clearLoaders = () => setLoaders({});
-  const stepLoader = useMemo(() => {
-    const data = Object.values(loaders).find(
-      (l) => (l as any)?.showOn === 'full_page'
-    );
-    if (!data) return null;
-    return (data as any).type === 'default' ? (
-      <div style={{ height: '20vh', width: '20vh' }}>
-        {(data as any).loader}
-      </div>
-    ) : (
-      (data as any).loader
-    );
-  }, [loaders]);
+  // When the active step changes, recalculate the dimensions of the new step
+  const stepCSS = useMemo(() => calculateStepCSS(activeStep), [activeStep]);
+
+  const { clearLoaders, loader, loaders, setLoaders } = useLoader({
```

Comment:
why does loaders still need to be returned here? isn't it not used now?

Reply:
I had removed it completely and then ran into a crash because it's used here - https://github.com/feathery-org/feathery-react/blob/master/src/Form/grid/Element/index.tsx#L184